frontend/src/components/cutscene/cutscene.component.js

```diff
@@ -19,20 +20,27 @@ const CutScene = () => {
 
     useEffect(() => {
     const timer = setTimeout(() => {
-      setBackground("bg-blue-500");
+      setBackground("bg-JapanCutscene");
     }, 3000);
     return () => clearTimeout(timer);
     }, []);
 
+    useEffect( () => {
+    const timer2 = setTimeout(() => {
+      setDisplayText(japan);
+    }, 5000);
+    return () => clearTimeout(timer2);
+    }, []);
+
 
 
     
 
   return (
-    <div className={`h-screen w-screen ${background}`}>
+    <div className={`h-screen w-screen bg-cover bg-center ${background}`}>
       <div className="flex justify-center items-center h-screen">
       <div
-        data-test="study-desk"
+        data-test="cut-scene"
         onClick={() => dispatch(setCurrentScene("BEDROOM"))}
         className="w-40 h-40 cursor-pointer overflow-hidden rounded-lg sm:rounded-sm bg-white bg-opacity-80 px-4 py-5 shadow sm:p-6 transform hover:translate-y-2 pixelated"
```

Comment:
maybe for hover, it could make the div larger rather than move? no worries if this is tricky. You could try scale-105 or manually edit the width and height on hover.